README.md

```diff
@@ -97,6 +85,53 @@ updates automatically.
 
 To further enhance your Yomitan experience, it's worth [integrating with Anki](./docs/anki-integration.md), a spaced-repetition flashcard program to help solidify the words you encounter.
 
+## Changes
+
+> While we have made some substantial changes, the majority of the extension's functionality is thanks to hard work of [FooSoft](https://foosoft.net/) and many other incredible open source contributors from 2016-2023.
+
+Major changes:
+
+- 💪 Completed the Manifest V2 → V3 transition, [read why here](https://developer.chrome.com/blog/resuming-the-transition-to-mv3/)!
+- ⚡ Switched to using ECMAScript modules and npm-sourced dependencies.
+- 🔧 Implemented an end-to-end CI/CD pipeline.
+- 🧪 Switched to standard testing frameworks, vitest and playwrights.
+
+In addition, we have made important bug fixes and minor enhancements:
+
+- ✨ Improved dictionary import speeds by 2x~10x.
+- 📚 Added functionality to import/export multiple dictionaries, enabling portability across devices.
+- 🕵️‍♂ And [more](https://github.com/themoeway/yomitan/pulls?q=is%3Apr+is%3Amerged+-label%3Aarea%2Fdependencies+-label%3Akind%2Fmeta).
```

Comment:
I don't think the detective emoji fits for this last bit and I think it would be better to use a book for the dictionary.
```suggestion
- 📖 Improved dictionary import speeds by 2x~10x.
- 📚 Added functionality to import/export multiple dictionaries, enabling portability across devices.
- ✨ And [more](https://github.com/themoeway/yomitan/pulls?q=is%3Apr+is%3Amerged+-label%3Aarea%2Fdependencies+-label%3Akind%2Fmeta).
```